model-gen/build/calm/2024-12/flow.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
The library seems to have a bug whereby just defs alone in a file will not generate any definitions. It will generate definitions via a schema which is why core.d.ts has flow definition generated.